.env.sample

```diff
@@ -0,0 +1,6 @@
+# Heroku Local
+RACK_ENV=development
+PORT=3000
+# Heroku Config Var
+YARN_PRODUCTION=false
+DATABASE_URL=postgres://iabuuyphkbuwis:9d9975c6d0a4eb19587e66fb4ce9a6787f6137d26f8e0a4200ed57e560550ab0@ec2-18-206-20-102.compute-1.amazonaws.com:5432/dbbgov5vn124re
```

Comment:
The purpose `.env.sample` file is to list all the names of the environment variables required in the app and not their values.

```
VARIABLE_1=
VARIABLE_2=
VARIABLE_3=
```

Reply:
Got it !